src/collectd.conf.pod

```diff
@@ -6625,6 +6633,29 @@ their configuration can be found in L<types.db(5)>.
 
 This optional setting sets the type instance to use.
 
+=item B<LatencyPercentile> I<Percent>
+
+Calculate and dispatch the configured percentile, i.e. compute the latency, so
+that I<Percent> of all matched latency values are smaller than or equal to the
+computed latency.
+
+Different percentiles can be calculated by setting this option several times.
+
+=item B<LatencyPercentileType> I<Type>
```

Comment:
Fixed type for reported values is not appropriate, it should be configurable.

The `tail` plugin `<Match>` block has `Type` option for reported values.
But with `DSType "Latency"` set I can report percentiles and rates both in one match block. 
That is typical task for latency - to report percentiles and rates both.

When both percentiles and rates are reported from one `Match` block,  rates must be sent as different type than percentiles.

So, two options was added: `LatencyPercentileType` and `LatencyRateType`.

When no value set by these options, then `Type` option is used in reported values. 
When both values are set by these options, then `Type` option is not used at all.

What do you think about these options?